openedx/adg/lms/applications/models.py

```diff
@@ -30,12 +36,43 @@ def __str__(self):
         return 'User {user_id}, application status id={id}'.format(user_id=self.user.id, id=self.id)
 
 
+class BusinessLine(TimeStampedModel):
+    """
+    Model to save the business lines offered by ADG.
+    """
+    title = models.CharField(max_length=255, )
+    logo = models.ImageField(
+        upload_to='business-lines/uploaded-logo/', null=True,
+        validators=[FileExtensionValidator(LOGO_ALLOWED_EXTENSION), validate_logo_size],
+    )
+    description = models.TextField()
+
+    def __str__(self):
+        return 'Business Line {}'.format(self.title)
+
+
 class UserApplication(TimeStampedModel):
     """
     Model for status of all required parts of user application submission.
     """
     user = models.OneToOneField(User, related_name='application', on_delete=models.CASCADE, )
     city = models.CharField(verbose_name=_('City'), max_length=255, )
+
+    saudi_national = models.BooleanField(default=True)
+    MALE = 'male'
+    FEMALE = 'female'
+    OTHER = 'other'
+
+    GENDER_CHOICES = (
+        (MALE, _('Male')),
+        (FEMALE, _('Female')),
+        (OTHER, _('Prefer not to answer')),
+    )
+    gender = models.CharField(choices=GENDER_CHOICES, max_length=8, default=OTHER,)
```

Comment:
Can you confirm `max_length=8` works for all choices especially when `OTHER` is selected

Reply:
Yes, works with other as well

Reply:
@javeriaejaz-arbisoft BTW we can add max_length=255 as well for CharFields where we don't need any validation for max_length beacuase in database it creates a varchar field which doesn't take extra memory. like it will take 8 bytes only even after setting max_length=255.